src/main.rs

```diff
@@ -32,6 +32,10 @@ mod args {
             release: bool,
             time: bool,
         },
+        Time {
+            day: Option<Day>,
+            force: bool,
```

Comment:
It think `--all` is more clear.
Generally `--force` is used to bypass a security check but not to change to behaviour of a command.